### DISCUSSION TITLE
Expose all virtual files on `_vfs`

### DISCUSSION THREAD

**Discussion by benjamincanac** [nitrojs/nitro#164] [Ideas]
Currently, I am using http://localhost:3000/_vfs/ a lot to understand what's happening when using [Nuxt](https://github.com/nuxt/framework) but it seems that only the `.nuxt/*` are exposed.

It would be really nice to expose all the virtual files with Nitro to better understand what's happening under the hood 😊

**Comment by pi0** [comment]
Thanks for the suggestion! Moved to https://github.com/unjs/nitro/issues/173
